test/integration/pdf_generation_test.exs

```diff
@@ -165,7 +165,7 @@ defmodule ChromicPDF.PDFGenerationTest do
 
     @tag :pdftotext
     test "it does not print PDFs from https:// URLs when given the offline: true parameter" do
-      assert_raise RuntimeError, ~r/net::ERR_INTERNET_DISCONNECTED/, fn ->
+      assert_raise ChromicPDF.ChromeError, ~r/net::ERR_INTERNET_DISCONNECTED/, fn ->
```

Comment:
would it maybe make sense to also assert that the error message contains the explanatory text?

Reply:
meh... could do, but there are ton's of newlines in there. I think I'm ok with asserting that it is a `ChromeError` :)

Reply:
I also thought about maybe the first sentence or so, just to make sure a comment is shown and not only the original error. But as you like :)

Reply:
Ok, done :+1:

TIL: `~r/foo.*bar/s` can match `"foo\n\n\nbar"`